Decisions.AdobeSign/Steps/AbstractStep.cs

```diff
@@ -9,11 +9,12 @@
 using DecisionsFramework.ServiceLayer.Services.ContextData;
 using System;
 using System.Collections.Generic;
+using DecisionsFramework.Design.Flow.CoreSteps;
 
 namespace Decisions.AdobeSign
 {
     [Writable]
-    public abstract class AbstractStep : ISyncStep, IDataConsumer, IValidationSource
+    public abstract class AbstractStep : BaseFlowAwareStep, ISyncStep, IDataConsumer, IValidationSource
```

Comment:
Did you test this on upgrade?

Reply:
You'll need to test by doing the following:

1. Uninstall and install to a brand new DB.
2. Install the module from features.
3. Build a flow that uses the steps in this module that extend `AbstractStep`.
4. Ensure the Outcome Scenario on the steps is not ignored, use Rename and name it something.
5. Build and deploy your module.
6. Ensure there are no issues with the steps or Outcome Scenarios.